SE ZipFileReader severce platfrom v2.1/MAIN.py

```diff
@@ -0,0 +1,103 @@
+import os, zipfile, pprint, time, datetime, sys
+import MAINCODE_Function_ZFR
+
+class MAIN_ZFR():
+
+
+    # main한 - this code is 한.
+    def main한():
```

Comment:
@choeminjun 
I'd like to write the function name using English otherwise it may cause unexpected error if the file is changed by other charset.